WooCommerce/Classes/ViewRelated/Dashboard/Settings/Themes/ThemeSettingViewModel.swift

```diff
@@ -12,19 +12,28 @@ final class ThemeSettingViewModel: ObservableObject {
     private let stores: StoresManager
     private let analytics: Analytics
     private(set) var carouselViewModel: ThemesCarouselViewModel
+    private let themeInstaller: ThemeInstallerProtocol
 
     init(siteID: Int64,
          stores: StoresManager = ServiceLocator.stores,
-         analytics: Analytics = ServiceLocator.analytics) {
+         analytics: Analytics = ServiceLocator.analytics,
+         themeInstaller: ThemeInstallerProtocol? = nil) {
```

Comment:
Same question as above about the optional installer

Reply:
Done in 36d0d57b7e121bbd4d9a3f7304c79d9871c07576